README.md

```diff
@@ -51,24 +53,35 @@ This project depends on Rust v1.65.0 or above and a modern C toolchain.
 The `llm` crate exports `llm-base` and the model crates (e.g. `bloom`, `gpt2`
 `llama`).
 
-To use `llm`, add it to your `Cargo.toml`:
+Add `llm` to your project by listing it as a dependency in `Cargo.toml`. To use
+the version of `llm` you see in the `main` branch of this repository, add it
+from GitHub (although keep in mind this is pre-release software):
 
 ```toml
 [dependencies]
-llm = "0.2"
+llm = { git = "https://github.com/rustformers/llm" , branch = "main" }
 ```
 
-**NOTE**: To improve debug performance, exclude `llm` from being built in debug
-mode:
+To use a [released](https://github.com/rustformers/llm/releases) version, add it
+from [crates.io](https://crates.io/crates/llm) by specifying the desired
+version:
 
 ```toml
-[profile.dev.package.llm]
+[dependencies]
+llm = "0.1"
+```
+
+**NOTE**: To improve debug performance, exclude the transitive `ggml-sys`
+dependency from being built in debug mode:
+
+```toml
+[profile.dev.package.ggml-sys]
 opt-level = 3
 ```
 
 ### Building `llm-cli`
 
-Follow these steps to build the command line application, which is named `llm`:
+Follow these steps to build the command-line application, which is named `llm`:
```

Comment:
Add a section here to download the CLI from Releases, with the caveat that it might not be the most up-to-date